### PR TITLE
Solved Issue Text overlap on coach > plan: manage resources section #10340

### PR DESCRIPTION
**Pull request by Pursottam6003**
## Summary

From the assigned Issue 
Text overlap on coach > plan: manage resources section #10340 Update the code section of **ResourcesListTable.vue**  

Now The resource section is responsive and there is no _**Text overlaps between the two rows**_

## References

[Kolibri Developers Documentation](https://kolibri-dev.readthedocs.io/en/develop/frontend_architecture/conventions.html)

## Reviewer guidance

Now we can see its working fine for 

- [X]  Mobile Devices
Mobile View

- [X]  Desktop View 

Desktop View

----

## Testing checklist

- [X] Contributor has fully tested the PR manually
- [X] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [X] PR has the correct target branch and milestone
- [X] PR has 'needs review' or 'work-in-progress' label
- [X] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by MisRob** [comment]
Thank you for your contribution @Pursottam6003. Could you please run the linter to ensure the linting check on the pull request passes?

**Comment by Pursottam6003** [comment]
> Thank you for your contribution @Pursottam6003. Could you please run the linter to ensure the linting check on the pull request passes?

Hii @MisRob  Mam could you please guide me l  how to pass the **lint test** actually, to be honest, I have no idea how to perform the lint test in a repository 
image

**Comment by MisRob** [comment]
@Pursottam6003 The most reliable way would be to install and run pre-commit as described here https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#pre-commit-hooks. Alternatively, there are some linting commands in package.json, however, I'd recommend pre-commit as it covers more than just JavaScript linting and corresponds to checks run on pull requests.

**Comment by Pursottam6003** [comment]
Hello @MisRob  done the linter  test 


Screenshot from 2023-04-03 18-30-21

Awaiting for merge  request to be approved

**Comment by thanksameeelian** [comment]
hi @Pursottam6003! i'd like to echo @MisRob and thank you again for the work you've done here 🙂

i'm really interested in your pull request & what you discovered while investigating the issue. would you be able to elaborate on what ended up being the issue with `KFixedGrid` and `KFixedGridItem` that was solved by your changes? (if your debugging led you in that direction)

whenever possible, we try to use components from our custom [Kolibri Design System](https://github.com/learningequality/kolibri-design-system/) (KDS), which includes [KFixedGrid](https://github.com/learningequality/kolibri-design-system/blob/develop/lib/grids/KFixedGrid.vue) & [KFixedGridItem](https://github.com/learningequality/kolibri-design-system/blob/develop/lib/grids/KFixedGridItem.vue). we welcome contributions in that repo as well, and i'd be interested to know if some of the improvements you've applied here could be incorporated as changes to those components in KDS instead, rather than removing the components from this part of Kolibri.

**Comment by Pursottam6003** [comment]
> hi @Pursottam6003! i'd like to echo @MisRob and thank you again for the work you've done here slightly_smiling_face
> 
> i'm really interested in your pull request & what you discovered while investigating the issue. would you be able to elaborate on what ended up being the issue with `KFixedGrid` and `KFixedGridItem` that was solved by your changes? (if your debugging led you in that direction)
> 
> whenever possible, we try to use components from our custom [Kolibri Design System](https://github.com/learningequality/kolibri-design-system/) (KDS), which includes [KFixedGrid](https://github.com/learningequality/kolibri-design-system/blob/develop/lib/grids/KFixedGrid.vue) & [KFixedGridItem](https://github.com/learningequality/kolibri-design-system/blob/develop/lib/grids/KFixedGridItem.vue). we welcome contributions in that repo as well, and i'd be interested to know if some of the improvements you've applied here could be incorporated as changes to those components in KDS instead, rather than removing the components from this part of Kolibri.

Hii @thanksameeelian  thanks for asking about  the issue actually while working with KFixedGrid I noticed it was quite difficult to understand the KFixedGrid and even the styles I was applying were not being applied on KFixedGridItems hence I had just changed with normal CSS flex 

I apologize but at first, I tried with KDS only  and I will again look into it in the future and not replace KDS 

Sincerely 
Pursottam Sah

**Comment by MisRob** [comment]
@Pursottam6003 There's no need to apologize. We just discussed this with a couple of members of the team, including @thanksameeelian, and there was an agreement that removing the grid is not such a big deal in this case. You can leave the pull request as is right now and we will review it soon. Thank you.

**Comment by MisRob** [comment]
@radinamatic A little QA would be welcome if you happened to have a moment, especially in Internet Explorer

**Comment by pcenov** [comment]
@MisRob I confirm - the text overlapping is fixed now in all supported browsers. Note that I am note able to check this in Internet Explorer 11 because of the following existing issue: https://github.com/learningequality/kolibri/issues/10086

**Comment by MisRob** [comment]
> @MisRob I confirm - the text overlapping is fixed now in all supported browsers. Note that I am note able to check this in Internet Explorer 11 because of the following existing issue: https://github.com/learningequality/kolibri/issues/10086

Thank you @pcenov.

If possible, my recommendation would be to wait before merging until we fix https://github.com/learningequality/kolibri/issues/10086 because due to some IE11 flexbox bugs, it's the browser where this needs to be tested the most.

File: kolibri/plugins/coach/assets/src/views/plan/LessonSummaryPage/ResourceListTable.vue [hunk 1]
```diff
@@ -10,12 +10,15 @@
         :key="resource.contentnode_id"
       >
         <DragHandle>
-          <KFixedGrid
+
+          <!-- replaced KFixedGrid with normal div to prevent text overlap error  -->
```
Comment (MisRob) [inline kolibri/plugins/coach/assets/src/views/plan/LessonSummaryPage/ResourceListTable.vue:14]: Helpful comments, thanks

**Review by MisRob — COMMENTED** [review]
When it comes to code, changes are looking good to me

File: kolibri/plugins/coach/assets/src/views/plan/LessonSummaryPage/ResourceListTable.vue (hunk 1 above)
Comment (Pursottam6003) [reply kolibri/plugins/coach/assets/src/views/plan/LessonSummaryPage/ResourceListTable.vue:14]: Thank you for your feedback mam

**Comment by thanksameeelian** [comment]
nice work @Pursottam6003 and thank you for explaining your thought process 😀 this looks great and is so helpful!

**Review by rtibbles — APPROVED** [review]
None of the used css properties are problematic in IE11, and the known bugs of display: flex in IE11 don't seem to be relevant here. See [here](https://caniuse.com/?search=display%3A%20flex) for more details.

**Comment by MisRob** [comment]
If my memory is correct, we've already seen a couple of issues with some flex-related styling like grow or shrink or perhaps some of their unhappy combination with another styling in the past in IE which was the basis for the recommendation to do some testing.

**Pull request merged**